services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainMetrics.java

```diff
@@ -185,6 +190,13 @@ public BeaconChainMetrics(
             "previous_epoch_total_weight",
             "Total effective balance of all active validators in the previous epoch");
 
+    headLiveSyncCommittee =
+        SettableGauge.create(
+            metricsSystem,
+            TekuMetricCategory.BEACON,
+            "head_live_synccommittee",
```

Comment:
is this an agreed name? do we want to add a `_` between sync and committee?

Reply:
No not an agreed name - I haven't heard of anyone thinking through what was needed for altair metrics yet.  And yes probably should have a _.